dapp/src/components/WelcomeModal/index.tsx

```diff
@@ -0,0 +1,179 @@
+import React from "react"
+import {
+  Button,
+  ModalBody,
+  ModalHeader,
+  Box,
+  ModalCloseButton,
+  Stepper,
+  Step,
+  StepIndicator,
+  ModalFooter,
+  useSteps,
+  SimpleGrid,
+  StepIndicatorProps,
+  UseStepsReturn,
+} from "@chakra-ui/react"
+import { H3, TextSm } from "#/components/shared/Typography"
+import { BaseModalProps } from "#/types"
+import withBaseModal from "../ModalRoot/withBaseModal"
+import step1Video from "../../assets/videos/step-1.mov"
```

Comment:
For ease of exports and clarity we can organize these video assets by purpose so `/assets/videos/welcome-steps/welcome-step-{n}.mov` with `index.ts` file. Please take a look at `/assets/images/benefits` as a reference.

Reply:
7da4627